src/api/custom-resolvers/mutations/account-sign-in/account-sign-in-checks/index.ts

```diff
@@ -0,0 +1,79 @@
+import confirmEmailAddressEmail from '../../../../helpers/send-email-confirm-email-address';
+import generateOTPAndUpdateAccount from '../../../../helpers/generate-otp-and-update-account';
+import getFullNameString from '../../../../helpers/get-full-name-string';
+import sendEmail from '../../../../emails';
+import { Account, Context } from '../../../../types';
+
+/**
+ * accountSignInChecks
+ * Assuming that a provided password is valid, check if the account is verified.
+ * If not, this means that the user has not verified their email address. We can therefore:
+ * 1) Reset the account's verification expiry.
+ * 2) Re-send the verification link email that was sent during account creation.
+ *
+ * Otherwise, the account is verified. We can therefore:
+ * 1) Generate an OTP
+ * 2) Update the account
+ * 3) Send an email with the OTP
+ *
+ * @param {Object} KeystoneJS context API
```

Comment:
Context? for param

Reply:
Updated all instances of this in the API 👍